includes/class-jws-functions.php

```diff
@@ -154,15 +154,15 @@ function jws_reading_time($word_count, $wpm = 250, $format = 'admin')
 
     if ($format == 'admin') {
         if ($minutes == 0) {
-            $html = '<1 minute';
+            $html = __( '<1 minute', 'just-writing-statistics' );
         } else {
             if ($hours == 0) {
-                $html = number_format($minutes).' minute'.($minutes > 1 ? 's' : '');
+                $html = number_format($minutes).__( ' minute(s)', 'just-writing-statistics' );
```

Comment:
Agree the if statement for 's' is incorrect, but the fix isn't '(s)', but instead to do two separate strings, one 'minute' and one 'minutes'.

Reply:
Agree.